CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Update build Dockerfile to create an entry in /etc/passwd for the user building
   the image.  It allows the `setup_jwt_cookie.sh` script to be run inside the
   container.
+* update Node version to 16.x in Docker file
```

Comment:
Does this correspond with a particular version of the cumulus dashboard?

Reply:
@reweeden I'm not sure that the dashboard uses Node.  It's all front-end stuff.  However if I look at an example lambda that Cumulus deploys, it does say node16:

https://github.com/nasa/cumulus/blob/master/tf-modules/s3-replicator/main.tf#L40

It appears the change happened going from 14.1:

https://github.com/nasa/cumulus/blob/v14.1.0/tf-modules/s3-replicator/main.tf

to 15.0

https://github.com/nasa/cumulus/blob/v15.0.0/tf-modules/s3-replicator/main.tf#L40

I personally never use node from within the Docker container. So, I'm not sure of the use case for having it in there.

Reply:
It's there in order to build the dashboard. We run `npm run build` which compiles the dashboard into a webpack distribution which can then be uploaded into an s3 bucket.

Reply:
It happens in the CIRRUS-DAAC makefile

Reply:
Looks like between v12.0.0 and v12.0.1 they bumped the node version in their `package.lock` file from 14 to 16.

Reply:
I actually run that `make dashboard` outside of the CIRRUS-core Docker because that make target runs in it's own own docker.  I couldn't get docker within docker to work.

Reply:
Ah, I see that I don't use the latest CIRRUS-DAAC make file.  Here's how I build my dashboard:

https://git.earthdata.nasa.gov/projects/ORNLCLOUD/repos/ornl-cumulus-core/browse/Makefile#243-291

Reply:
@reweeden CHANGELOG updated.  If it looks ok, feel free to merge.